spec/models/queue_item_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+
+describe QueueItem do 
+  it { should belong_to(:user)}
+  it { should belong_to(:video)}
+
+  describe '#video_title' do 
```

Comment:
You could eventually refactor this with should matchers :) Check [this](https://github.com/thoughtbot/shoulda-matchers/blob/master/lib/shoulda/matchers/independent/delegate_method_matcher.rb) out.